pkg/bitcoin/electrum/electrum.go

```diff
@@ -566,6 +566,38 @@ func (c *Connection) getScriptMempool(
 	return convertedItems, nil
 }
 
+// EstimateSatPerVByteFee returns the estimated sat/vbyte fee for a
+// transaction to be confirmed within the given number blocks.
+func (c *Connection) EstimateSatPerVByteFee(blocks uint32) (int64, error) {
+	// According to Electrum protocol docs, the returned fee is BTC/KB.
+	btcPerKbFee, err := requestWithRetry(
+		c,
+		func(
+			ctx context.Context,
+			client *electrum.Client,
+		) (float32, error) {
+			return client.GetFee(ctx, blocks)
```

Comment:
The `client.GetFee` is part of the `go-electrum` library. IIRC, the docstring of this function points to the protocol docs anyway.

Reply:
Correct, I was hoping for a PR to our fork: https://github.com/keep-network/go-electrum

Eventually, I think we should [drop the replacement rule](https://github.com/keep-network/keep-core/blob/5bb7d524e4124fcb1cfc42ee7dab75915bba6228/go.mod#L11) and switch to our fork. I don't think the referenced library is actively maintained.

The docstring refers to the protocol docs but this information is quite important and I'd love to have it in the Go documentation as well, especially since it mentions [the fee is per kilobytes](https://github.com/keep-network/go-electrum/blob/567f2c1de4bc694a7b864b04944f86fc2018f08d/electrum/misc.go#L14-L17). I do not consider it a blocker but a really-nice-to-have followup.